cache.go

```diff
@@ -135,6 +135,17 @@ func (c *cache) Get(k string) (interface{}, bool) {
 	return item.Object, true
 }
 
+//GetMultipleItems returns an array of items corresponding to the input array
+func (c *cache) GetMultipleItems(values []string) []interface{} {
+	length := len(values)
+	var items = make([]interface{}, length)
```

Comment:
i would consider if the length of values is 0 an inmediate return, we could be initializing a 0 length map otherwise